### DISCUSSION TITLE
Using Helm package tgz compressed tarball in CNTI tests

### DISCUSSION THREAD

**Discussion by sysarch-repo** [lfn-cnti/testsuite#2041] [General]
Is there a possibility to reference a Helm package compressed tarball (tgz) in the cnf-testsuite.yml config file?
Currently, it seems that the helm_directory parameter must point to a location that contains Chart.yaml, instead of a tgz file.
Or is there a different parameter to be used in this context?

**Comment by martin-mat** [comment]
helm_directory parameter takes also a tarball, like:
```
helm_directory: external-dns-7.2.0.tgz
```
I checked that and working for me.

**Comment by sysarch-repo** [comment]
@martin-mat Works nicely. Thanks a lot!
